doc/crypto/RAND_add.pod

```diff
@@ -42,25 +42,15 @@ or L<RAND_load_file(3)>.
 
 RAND_seed() is equivalent to RAND_add() when B<num == entropy>.
 
-RAND_event() collects the entropy from Windows events such as mouse
-movements and other user interaction. It should be called with the
-B<iMsg>, B<wParam> and B<lParam> arguments of I<all> messages sent to
-the window procedure. It will estimate the entropy contained in the
-event message (if any), and add it to the PRNG. The program can then
-process the messages as usual.
-
-The RAND_screen() function is available for the convenience of Windows
-programmers. It adds the current contents of the screen to the PRNG.
-For applications that can catch Windows events, seeding the PRNG by
-calling RAND_event() is a significantly better source of
-randomness. It should be noted that both methods cannot be used on
-servers that run without user interaction.
-
 =head1 RETURN VALUES
 
-RAND_status() and RAND_event() return 1 if the PRNG has been seeded
+RAND_status() returns 1 if the PRNG has been seeded
 with enough data, 0 otherwise.
 
+RAND_event() calls RAND_poll() and returns RAND_seed().
+
+RAND_screen calls RAND_poll().
+
 The other functions do not return values.
 
 =head1 SEE ALSO
```

Comment:
Please can you add an HISTORY section stating that RAND_screen() and RAND_event() were deprecated in 1.1.0 (see other pod files for an example).